clients/input_copy.go

```diff
@@ -155,3 +153,7 @@ type StubInputCopy struct{}
 func (s *StubInputCopy) CopyInputToS3(requestID string, inputFile, osTransferURL *url.URL) (inputVideoProbe video.InputVideo, signedURL string, err error) {
 	return video.InputVideo{}, "", nil
 }
+
+func RetryBackoff() backoff.BackOff {
+	return backoff.WithMaxRetries(newExponentialBackOffExecutor(), 5)
```

Comment:
Maybe better to create two functions? 
`func RetryExponentialBackoff()` and `func RetryConstantBackoff()`